client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss

```diff
@@ -1,20 +1,19 @@
-.InfiniteScrollTable {
+.InfiniteScrollTable,
+/* WARNING: Do NOT use `FiniteScrollTable`. Instead, implement `InfiniteScrollTable` */
+.FiniteScrollTable {
   --cell-horizontal-padding: 0.35em; /* horizontal cell padding for inter-column buffer */
-  /* TODO: After, FP-103, use `composes:` not `@extend` */
-  @extend .o-fixed-header-table;
```

Comment:
__Why Remove This?__ Because:

1. `.o-fixed-header-table` is global, and app is not configured to support `@extend`ing global classes.
2. CSS Modules offers `composes: ` (which can be used after `InfiniteScrollTable` uses CSS Modules).
3. `@extend` is full of caveats (see "Heads up!" in [SASS documentation](https://sass-lang.com/documentation/at-rules/extend)) making it no more simpler than `composes:`.
4. We should only use one means to import/extend/compose CSS, and `composes:` is the choice (see step 2).